packages/ui/components/ui/tx/view/value.tsx

```diff
@@ -36,8 +37,8 @@ export const ValueViewComponent = ({ view }: ValueViewPrpos) => {
     const value = view.valueView.value;
     const amount = value.amount ?? new Amount();
     const display_denom = value.denom?.display ?? '';
-    // The first denom unit in the list is the display denom, according to cosmos practice
-    const exponent = value.denom?.denomUnits[0]?.exponent ?? 1;
+    const exponent = value.denom ? getDisplayDenomExponent(value.denom) : 0;
```

Comment:
Same issue here, changing the exponent from 1 to 0.

Also, the comment which I deleted says:
> The first denom unit in the list is the display denom, according to cosmos practice

However, I'm not sure this is correct. Cosmos's [own ADR docs](https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-024-coin-metadata.md) contain an example asset where the first denom is _not_ the display denom. So, to be safe, I'm calling `getDisplayDenomExponent` to make sure we're getting the right exponent.

Reply:
i agree it's preferable to certainly identify it rather than assume, especially since it doesn't seem to be an expensive search